tests/auto/mocks/qmozsecurity/qmozsecurity.h

```diff
@@ -0,0 +1,31 @@
+/****************************************************************************
+**
+** Copyright (C) 2015 Jolla Ltd.
+** Contact: Piotr Tworek <piotr.tworek@jolla.com>
```

Comment:
Wrong copy rights

Reply:
Thanks, I've updated all the copyright headers for the files I touched.